airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -502,9 +500,9 @@ public WebBackendConnectionRead webBackendUpdateConnection(final WebBackendConne
    * Given a fully updated connection, check for a diff between the old catalog and the updated
```

Comment:
This doc looks inaccurate.

Reply:
oh, it looks like we're currently calculating the diff twice in this endpoint. i'll update to use the existing diff.

Reply:
actually - this is in a separate endpoint, and it is calculating the diff between the existing sync catalog & new sync catalog, so I think this doc is accurate